### PR TITLE
Changes on the Dockerfile:

### PR DESCRIPTION
**Pull request by agpenton**
All the changes below will resolves geosolutions-it/C207-OSM#6
* Base image updated.
* PostgreSQL-client updated to version 11.
* PBF file changed to Sweden.

### CODE REVIEW

File: docker-imposm3/Dockerfile [hunk 1]
```diff
@@ -1,9 +1,10 @@
-FROM golang:1.10
+FROM golang:buster
```
Comment (randomorder) [inline docker-imposm3/Dockerfile]: please use a fixed version number instead

File: docker-imposm3/Dockerfile [hunk 2]
```diff
@@ -18,5 +19,12 @@ RUN pip3 install -r /home/requirements.txt
 
 ADD importer.py /home/
 
+# Add YAML Mapping file
+ADD mapping.yml /home/settings/mapping.yml
```
Comment (randomorder) [inline docker-imposm3/Dockerfile:23]: we may need to update this with the one in use for the Maps server: https://github.com/geosolutions-it/osm-styles/blob/master/imposm/mapping.yml

**Review by randomorder — CHANGES_REQUESTED** [review]

File: docker-imposm3/Dockerfile (hunk 1 above)
Comment (agpenton) [reply docker-imposm3/Dockerfile]: base image changed in the last commit 483fd3e

File: docker-imposm3/Dockerfile (hunk 2 above)
Comment (agpenton) [reply docker-imposm3/Dockerfile:23]: the mapping file is changed in the last commit 483fd3e  with the referred above.

**Pull request merged**